features/writing-mode-deprecated.yml

```
@@ -0,0 +1,16 @@
+name: Writing mode (deprecated)
+description: The `lr`, `lr-tb`, `rl`, `rl-tb`, `tb`, and `tb-rl` keyword values represent writing modes for elements in SVG 1.1 documents.
```

Let's line this up closer to the other writing-mode features

```suggestion
name: writing-mode SVG 1.1 values
description: The `lr`, `lr-tb`, `rl`, `rl-tb`, `tb`, and `tb-rl` keyword values for the `writing-mode` CSS property set the text direction for elements in SVG 1.1 documents.
```